eng/common/scripts/Helpers/PSModule-Helpers.ps1

```diff
@@ -47,6 +46,65 @@ function Update-PSModulePathForCI()
   }
 }
 
+function Get-ModuleRepositories([string]$moduleName) {
+  $DefaultPSRepositoryUrl = "https://www.powershellgallery.com/api/v2"
+  # List of modules+versions we want to replace with internal feed sources for reliability, security, etc.
+  $packageFeedOverrides = @{
+    'powershell-yaml' = 'https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-tools/nuget/v2'
+  }
+
+  $repoUrls = if ($packageFeedOverrides.Contains("${moduleName}")) {
+    @($packageFeedOverrides["${moduleName}"], $DefaultPSRepositoryUrl)
+  } else {
+    @($DefaultPSRepositoryUrl)
+  }
+
+  return $repoUrls
+}
+
+function moduleIsInstalled([string]$moduleName, [string]$version) {
```

Comment:
nit:
```suggestion
function ModuleIsInstalled([string]$moduleName, [string]$version) {
```

Reply:
I marked it lowercase as a sort of "private" function, given this is a sourced file only.